frontend/tests/features/cellCards/cellCards.test.ts

```diff
@@ -420,35 +448,36 @@ describe("Cell Guide", () => {
           `${TEST_URL}${ROUTES.CELL_GUIDE}/${NEURON_CELL_TYPE_ID}`,
           page
         );
+
         await page
           .getByTestId(CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW)
           .waitFor({ timeout: 5000 });
+
         const nodesLocator = `[data-testid^='${CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW_RECT_OR_CIRCLE_PREFIX_ID}']`;
+
         // collapse node's children
-        const nodesBefore = await page.locator(nodesLocator).elementHandles();
+        const nodesBefore = await page.locator(nodesLocator).all();
```

Comment:
Playwright prefers `all()` over `elementHandles()`:

> Playwright: **NOTE** Always prefer using [Locator]s and web assertions over [ElementHandle]s because latter are inherently racy.